writer/test-writer.ts

```diff
@@ -0,0 +1,29 @@
+import { Chroma } from "@langchain/community/vectorstores/chroma";
+import { OpenAIEmbeddings } from "@langchain/openai";
+import path from 'path';
+import dotenv from 'dotenv';
+
+// Load environment variables from .env file
+dotenv.config({ path: path.resolve(__dirname, '../.env') });
```

Comment:
Can u maybe write a small README of what needs to be configured in the .env

Reply:
Yes, will do that in another PR

Reply:
ok